src/theme/footer.php

```diff
@@ -1,96 +1,98 @@
 <?php
-  /**
-   * Copyright 2017 Google Inc. All Rights Reserved.
-   * Licensed under the Apache License, Version 2.0 (the "License");
-   * you may not use this file except in compliance with the License.
-   * You may obtain a copy of the License at
-   *     http://www.apache.org/licenses/LICENSE-2.0
-   * Unless required by applicable law or agreed to in writing, software
-   * distributed under the License is distributed on an "AS IS" BASIS,
-   * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   * See the License for the specific language governing permissions and
-   * limitations under the License.
-   */
-?>
-<?php
-  if ( defined( 'ABSPATH' ) )
-    require_once( ABSPATH . '/wp-load.php' );
-  etag_start();
-?>
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+  exit;
+}
+
+require_once ABSPATH . '/wp-load.php';
+
+$template_url = get_bloginfo( 'template_url' );
```

Comment:
`get_template_directory_uri()` would be the correct function

Reply:
Yes, is more correct.

This function was already on the theme and I kept it.
https://github.com/GoogleChromeLabs/ProgressiveWordPress/blob/master/src/theme/footer.php#L27

If we look inside the function `get_bloginfo( 'template_url' )` it's return the same ( `get_template_directory_uri()` ).
https://developer.wordpress.org/reference/functions/get_bloginfo/#source
Line: 695

Reply:
I mentioned it because you already where changing the code. `get_bloginfo()` is a really old function and is not coded following the best modern coding practices so it is possible that it may get depreciated in the future.

Reply:
Yes, I agree with you.

Reply:
For the record: I really appreciate both of you bringing my mostly-googled WP knowledge up-to-speed 🎉